Gems/Atom/RPI/Code/Source/RPI.Public/RPISystem.cpp

```diff
@@ -95,8 +95,8 @@ namespace AZ
                 }
             }
 
-            //Init RHI device
-            m_rhiSystem.InitDevice();
+            // Init RHI device(s)
+            m_rhiSystem.InitDevices(AzFramework::StringFunc::Equal(RHI::GetCommandLineValue("rhi-multiple-devices").c_str(), "enable"));
```

Comment:
I am not sure what the best name is, the multi-device functionality is still implemented on the `RHI` level, so I would either prefer keeping it as `rhi-multiple-devices` or change it to something more generic.

Reply:
How about simply "enableMultipleDevices"? rhi or rpi prefix is not important as really what we are trying to convey here is that we want to use all local physical devices if possible. To truly enable multiple devices we will need pipelines at RPI level so really we need both RPI and RHI to support multiple devices for the end user.

Reply:
I would imagine that in future this command line prompt may change to specify number of max devices to enable. So something along the lines of -numMultipleDevices=2

Reply:
This might depend a bit on how we want to specify the number and selection of devices later on, a mask might make sense, but for the mean time I would suggest we stick with `enableMultipleDevices` and we can revisit this later once we actually can assign passes to different devices.